docs/release-notes/platform_10-9-0_releasenotes.md

```diff
@@ -0,0 +1,117 @@
+---
+id: v10.9.0
+title: Version 10.9.0 Release Notes
+sidebar_label: v10.9.0
+image: "img/release-note-link-preview.png"
+---
+
+_May 04, 2023_
+
+## Console
+
+### New Features
+
+#### Selection of default providers for the Company's projects
+
+Within the Providers section in the Company Overview area it is now possible to find a new feature that allows the user with Company Owner permissions to be able to set the Default providers that will use all the new projects created on the respective company.
+
+![Default Providers Selection](./img/v10.9_defaultProviders.png)
+
+This will enhance the governance for every Company Owner for company's projects and give better control over the management of company tools.
+The provider for each capability (Git Provider, CI/CD and Secret Manager) can always be changed in any moment via CMS.
```

Comment:
```suggestion
This will enhance the governance of Company Owners on all the Projects of their Companies, granting them better control over the management of company tools.
The Provider for each capability (Git Provider, CI/CD and Secret Manager) can always be changed at any moment via CMS.
```